cli/tests/default/e2e/test_osemgrep_output.py

```diff
@@ -8,14 +13,33 @@
 @pytest.mark.parametrize("output_format", [OutputFormat.SARIF])
 @pytest.mark.parametrize(
     "rule_and_target",
-    [("rules/eqeq.yaml", "basic/stupid.py"), ("rules/cwe_tag.yaml", "basic/stupid.py")],
+    [
+        # Simple case that should pass.
+        ("rules/eqeq.yaml", "basic/stupid.py"),
+        # Whenever there's a CWE tag, there should be a security tag.
+        ("rules/cwe_tag.yaml", "basic/stupid.py"),
+        # Rules with metavariable-type need parser initialization to parse correctly.
+        ("rules/metavariable_type.yaml", "basic/stupid.py"),
+        # Dataflow traces in SARIF should abide by --dataflow-traces.
+        ("rules/taint_trace.yaml", "taint/taint_trace.cpp"),
+    ],
 )
-def test_sarif_output(run_semgrep_in_tmp: RunSemgrep, output_format, rule_and_target):
+@pytest.mark.parametrize("dataflow_traces", [True, False])
```

Comment:
so this test is also passing when it's osemgrep that is run right?
I mean running osemgrep directly, because in that case I assume osemgrep will just ignore the --use-osemgrep-sarif flag as it's already osemgrep :)

there is no need for @osemfail here right?

Reply:
Correct. No need for osemfail here.